api/util/util.go

```diff
@@ -21,3 +21,21 @@ func CloneMap(original map[string]string) map[string]string {
 	}
 	return m
 }
+
+// MergeLabels merges two given labels
+func MergeLabels(l ...map[string]string) map[string]string {
```

Comment:
This util function is copied from https://github.com/banzaicloud/koperator/blob/master/pkg/util/util.go#L104 to avoid the import cycle, not sure if it's better to delete the original function and make all the references of the original function to this copied function

Reply:
We should delete the original one.

Reply:
ok will do